README.md

```diff
@@ -151,6 +153,24 @@ Now you have ability to disable elements of Spaceship. All options must be overr
 | `SPACESHIP_RUBY_SHOW` | `true` | Current Ruby version |
 | `SPACESHIP_RUBY_SYMBOL` | `💎` | Character to be shown before Ruby version |
 
+### Xcode
+
+| Variable | Default | Meaning |
+| :------- | :-----: | ------- |
+| `SPACESHIP_SWIFT_SHOW_LOCAL` | `true` | Current local Swift version based on [swiftenv](https://github.com/kylef/swiftenv) |
+| `SPACESHIP_SWIFT_SHOW_GLOBAL` | `false` | Global Swift version based on [swiftenv](https://github.com/kylef/swiftenv) |
+| `SPACESHIP_XCODE_SYMBOL` | `🐦` | Character to be shown before Swift version |
+
+
+### Swift
+
+| Variable | Default | Meaning |
+| :------- | :-----: | ------- |
+| `SPACESHIP_SWIFT_SHOW_LOCAL` | `true` | Current local Xcode version based on [xcenv]() |
+| `SPACESHIP_SWIFT_SHOW_GLOBAL` | `true` | Global Xcode version based on [xcenv]() |
+| `SPACESHIP_SWIFT_SYMBOL` | `🛠` | Character to be shown before Xcode version |
+
+
```

Comment:
BTW, don't forget to add these variables to [settings' boilerplate](https://github.com/denysdovhan/spaceship-zsh-theme/#example) below.